app/backend/src/couchers/servicers/communities.py

```diff
@@ -305,10 +308,7 @@ def JoinCommunity(self, request, context):
                 context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.ALREADY_IN_COMMUNITY)
 
             node.official_cluster.cluster_subscriptions.append(
-                ClusterSubscription(
-                    user_id=context.user_id,
-                    role=ClusterRole.member,
-                )
+                ClusterSubscription(user_id=context.user_id, role=ClusterRole.member,)
```

Comment:
Please run the auto-formatters in `//app/backend` (so it picks up the config)

Reply:
When I run this in `//ap/backend` it reformats a bunch of files. I run the following command as stated in the documentation.

`autoflake -r -i --remove-all-unused-imports src && isort . && black . in the //app/backend`

Reply:
Which ones?

Reply:
``` 
        modified:   src/couchers/email/__init__.py
        modified:   src/couchers/jobs/enqueue.py
        modified:   src/couchers/jobs/handlers.py
        modified:   src/couchers/jobs/worker.py
        modified:   src/couchers/migrations/env.py
        modified:   src/couchers/migrations/versions/128252798bc4_add_new_email_pattern_constraint.py
        modified:   src/couchers/migrations/versions/3b8d963e0b7d_regions_and_languages.py
        modified:   src/couchers/migrations/versions/c035cdcabd40_add_new_events_tables.py
        modified:   src/couchers/models.py
        modified:   src/couchers/server.py
        modified:   src/couchers/servicers/account.py
        modified:   src/couchers/servicers/api.py
        modified:   src/couchers/servicers/auth.py
        modified:   src/couchers/servicers/blocking.py
        modified:   src/couchers/servicers/conversations.py
        modified:   src/couchers/servicers/events.py
        modified:   src/couchers/servicers/groups.py
        modified:   src/couchers/servicers/jail.py
        modified:   src/couchers/servicers/media.py
        modified:   src/couchers/servicers/pages.py
        modified:   src/couchers/servicers/requests.py
        modified:   src/couchers/servicers/search.py
        modified:   src/couchers/servicers/threads.py
        modified:   src/couchers/tasks.py
        modified:   src/dummy_data.py
        modified:   src/tests/test_account.py
        modified:   src/tests/test_api.py
        modified:   src/tests/test_auth.py
        modified:   src/tests/test_bg_jobs.py
        modified:   src/tests/test_communities.py
        modified:   src/tests/test_discussions.py
        modified:   src/tests/test_email.py
        modified:   src/tests/test_events.py
        modified:   src/tests/test_fixtures.py
        modified:   src/tests/test_groups.py
        modified:   src/tests/test_jail.py
        modified:   src/tests/test_pages.py
        modified:   src/tests/test_references.py
        modified:   src/tests/test_search.py
```

Reply:
If you're still having this issue, try entering backend then running it. I think it's grabbing its settings from app (which has no settings, so it's using default) instead of from backend, where we use length 120.